pg_search/src/postgres/options.rs

```diff
@@ -717,8 +750,8 @@ pub unsafe fn init() {
     );
     pg_sys::add_string_reloption(
         RELOPT_KIND_PDB,
-        "layer_sizes".as_pg_cstr(),
-        "The sizes of each segment merge layer".as_pg_cstr(),
+        "foreground_layer_sizes".as_pg_cstr(),
+        "The sizes of each layer to merge in the foreground".as_pg_cstr(),
```

Comment:
Again, lets keep the key as `layer_sizes`.  The new description is fine, of course.